ui/src/routes/runs/$runId/index.tsx

```diff
@@ -49,7 +50,11 @@ export const Route = createFileRoute('/runs/$runId/')({
       });
     }
   },
-  notFoundComponent: () => {
-    return <div>ORT run not found!</div>;
+  errorComponent: ({ error }) => {
+    if (error instanceof NotFoundError) {
+      return <>Sorry, no run exists with that ID.</>;
```

Comment:
Ideally, we'd mention the run ID as part of the error message: Like "There is no run with ID 666."